gateway/gateway.proto

```diff
@@ -76,4 +76,17 @@ message PreparedTransaction {
 // This is currently experimental and highly likely to change during gateway development.
 message Event {
     bytes value = 1;
+}
+
+// EndpointError contains details of errors that are received by any of the endorsing peers
+// as a result of processing the Evaluate or Endorse services, or from the ordering node(s) as a result of
+// processing the Submit service.
+// If any of these services return an error, then these details can be extracted using the Details() method.
+message EndpointError {
+    // The address of the endorsing peer or ordering node that returned an error.
+    string address = 1;
+    // The mspid of this endpoint
+    string mspid = 2;
```

Comment:
All other field names in this package that include `id` use the snake_case to separate the stem from the suffix. Should this be consistent with that pattern?